catalog/ui/catalog-ui-search/src/main/webapp/component/property/property.collection.view.js

```diff
@@ -216,23 +216,16 @@ define([
                 metacards.forEach(function(metacard) {
                     var value = metacard[property.id];
                     var isDefined = value !== undefined;
+                    let hasConflictingDefinition = false;
                     if (isDefined) {
                         if (!metacardDefinitions.metacardTypes[property.id].multivalued){
-                            if (value.sort === undefined){
+                            if (value.constructor !== Array){
```

Comment:
Are we trying to determine if the value is an array? Would `Array.isArray` be better?